addons/account/views/partner_view.xml

```diff
@@ -224,6 +224,10 @@
                                     <field name="credit_limit" class="oe_inline" invisible="not use_partner_credit_limit"/>
                                 </div>
                             </group>
+                            <group groups="base.group_no_one">
```

Comment:
- Are we sure we want this in debug mode?
- Shouldn't we add another group too? It feels like a random newbie accountant could enter the debug mode and change that parameter